fedmsg_meta_fedora_infrastructure/tests/coprs.py

```diff
@@ -249,6 +249,39 @@ class TestCoprsWorkerCreate(Base):
     }
 
 
+class TestCoprsGroupLink(Base):
+    """ `Copr <https://fedorahosted.org/copr/>`_ publishes these messages
+    when a build has completed.  This is an example of a group repo.
+    """
+    expected_link = ("https://copr.fedoraproject.org/coprs/"
```

Comment:
Indeed, the title isn't really informative, the subtitle is though: 
https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/blob/develop/fedmsg_meta_fedora_infrastructure/tests/coprs.py#L103-L104